manifest_staging/charts/gatekeeper/templates/gatekeeper-manager-role-clusterrole.yaml

```diff
@@ -17,6 +17,7 @@ rules:
   verbs:
   - get
   - list
+  - update
```

Comment:
Do we need the ability to modify all resources in the cluster yet?

Reply:
Removed,
We probably don't need this explicitely for the webhook